python/foglamp/services/core/service_registry/monitor.py

```diff
@@ -44,8 +44,11 @@ def __init__(self):
         self._ping_timeout = None  # type: int
         """Timeout for a response from any given micro-service"""
 
+    async def _sleep(self, sleep_time):
+        await asyncio.sleep(sleep_time)
+
     async def _monitor_loop(self):
-        """Main loop for the scheduler"""
+        """async Monitor loop to monitor registered services"""
         # check health of all micro-services every N seconds
         _MAX_ATTEMPTS = 15
         """Number of max attempts for finding a heartbeat of service"""
```

Comment:
I would suggest to make it module level variable, so that in *test* we can set to 2 or 3 and minimize the loop time.

Reply:
We can make it part of the configuration as well. But let's consider it during refactoring rather than now.